Dockerfile

```diff
@@ -9,15 +9,13 @@ RUN apk add --update \
     python \
     py-pip \
     nodejs \
-    ruby \
+    git \
```

Comment:
This is required by the node-sass build, which builds the native libsass C++ lib as part of its install process